packages/core/useStorage/index.ts

```diff
@@ -11,23 +11,23 @@ export type Serializer<T> = {
 
 export const StorageSerializers: Record<'boolean' | 'object' | 'number' | 'any' | 'string', Serializer<any>> = {
   boolean: {
-    read: (v: any) => v != null ? v === 'true' : null,
+    read: (v: any) => v === 'true',
     write: (v: any) => String(v),
   },
   object: {
-    read: (v: any) => v ? JSON.parse(v) : null,
+    read: (v: any) => JSON.parse(v),
     write: (v: any) => JSON.stringify(v),
   },
   number: {
-    read: (v: any) => v != null ? Number.parseFloat(v) : null,
+    read: (v: any) => Number.parseFloat(v),
     write: (v: any) => String(v),
   },
   any: {
-    read: (v: any) => (v != null && v !== 'null') ? v : null,
+    read: (v: any) => v,
     write: (v: any) => String(v),
   },
   string: {
-    read: (v: any) => v != null ? v : null,
+    read: (v: any) => v,
     write: (v: any) => String(v),
```

Comment:
@jeffrey-dot-li if the user passes `null` wouldn't this still convert it to a string of `"null"`? or am I missing something?

Reply:
Nah if the use passes `null` it doesn't call `serializer.write` in the first place so there is no string conversion - line 115:
```
        if (defaultValue !== null)
          storage.setItem(key, serializer.write(defaultValue))
```